packages/react/src/components/ProgressIndicator/ProgressIndicator-story.js

```diff
@@ -16,14 +16,25 @@ import { settings } from 'carbon-components';
 
 const { prefix } = settings;
 
+let spaceEquallyKnob = () => {
+  const spaceEqually = boolean('Space Equally', false);
+  return spaceEqually;
+};
+
+let verticalKnob = () => {
+  const vertical = boolean('Vertical', false);
+  return vertical;
+};
+
 storiesOf('ProgressIndicator', module)
   .addDecorator(withKnobs)
   .add(
     'Default',
     () => (
       <ProgressIndicator
-        vertical={boolean('Vertical', false)}
-        currentIndex={number('Current progress (currentIndex)', 1)}>
+        vertical={verticalKnob()}
```

Comment:
One tiny comment, is there any benefit to adding the knob this way? Otherwise, it may be nice to be consistent through our stories and just keep it as `vertical={boolean('Vertical', false)}`